### PR TITLE
Project guess who game

### PR DESCRIPTION
**Pull request by dannuzak**

### CODE REVIEW

**Review by NoemiSalome — APPROVED** [review]
Hi Daniela!
Thanks for letting me go through your code! Bit nervous about this to be honest. 
Great job with this project! This was a hard one, right? You came up with a different approach than I did and I just followed the steps in the lecture (and felt a lot like i was cheating:)) You came up with something else, which is great!
I would have loved to use some nice emojis in my comments as our codecoaches do, but couldn't find them. So just imagine a thumbs-up, unicorn and a dolphin every now and then:)
I hope I don't confuse you with my comments too much. If you end up wondering "what the heck does she mean" just let me know and we can meet in the foxes room ;)
Take care
Noemi

File: code/script.js [hunk 1]
```diff
@@ -254,90 +260,176 @@ const start = () => {
...
+  filterCharacters(willKeep)
 }
 
 // It'll filter the characters array and redraw the game board.
 const filterCharacters = (keep) => {
+  console.log(keep)
+
+  const group = currentQuestion.category
+  const attribute = currentQuestion.attribute
+  const physicalAttribute = currentQuestion.value
```
[the remark this answers is not in the document]
Comment (dannuzak) [reply code/script.js:341]: Great to know, I didn't know that, I saw it in Estefania's code but didn't know what it meant. Now I know, thanks, Noemi! But to be honest, I had a difficult time with the different names of properties of the object that I kind of helped myself having it this way. But appreciate learning something new, thanks!

File: code/script.js [hunk 2]
```diff
@@ -254,90 +260,176 @@ const start = () => {
...
   }
+  
 }
 
 // This function should be invoked when you click on 'Find Out'.
 const checkQuestion = () => {
   // Compare the currentQuestion with the secret person.
   // See if we should keep or remove people based on that
   // Then invoke filterCharacters
+  selectQuestion()
```
[the remark this answers is not in the document]
Comment (dannuzak) [reply code/script.js:317]: I was getting an error without it.  I couldn't really understand why the value couldn't be read eventho I tried saving it in a const variable. The error disappeared invoking this function right there.

File: code/script.js [hunk 3]
```diff
@@ -254,90 +260,176 @@ const start = () => {
...
+    willKeep = false
+  }
+
+  // Then invoke filterCharacters
+  filterCharacters(willKeep)
 }
 
 // It'll filter the characters array and redraw the game board.
 const filterCharacters = (keep) => {
+  console.log(keep)
```
[the remark this answers is not in the document]
Comment (dannuzak) [reply code/script.js:337]: oopsie, yeah, I know.

File: code/script.js [hunk 4]
```diff
@@ -254,90 +260,176 @@ const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
   // What else should happen when we start the game?
+
+  //Displaying the board so we can start the game
```
[the remark this answers is not in the document]
Comment (dannuzak) [reply code/script.js:264]: True that, thanks for the reminder!

File: README.md [hunk 5]
```diff
@@ -1,13 +1,17 @@
 # Project Name
```
[the remark this answers is not in the document]
Comment (dannuzak) [reply README.md:1]: hahaha, definitely!

File: code/script.js [hunk 6]
```diff
@@ -254,90 +260,176 @@ const start = () => {
...
+  if (secretAttribute === currentQuestion.value) {
+    // See if we should keep or remove people based on that
+    willKeep = true
+  }
+  else {
+    willKeep = false
+  }
+
+  // Then invoke filterCharacters
+  filterCharacters(willKeep)
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/script.js:332]: code from line 320 to 332 could be written in one line:
`filterCharacters(secretAttribute === currentQuestion.value)`

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 I admire that you respond to the comments left to you by your friend! it is important to communicate and fix errors pointed by others 🦄 

I know it was an extra point of this assignment to style it your way but remember that there is no another way to master CSS then practice it over and over again 🦾 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**